### DISCUSSION TITLE
Using a slider to control the zoom function

### DISCUSSION THREAD

**Discussion by upswing1** [Live-Charts/LiveCharts2#1228] [General]
My charts are inside a scrollview so I need to control the zooming function not with the mouse wheel as done in windows 11.  How can I do that with a slider?

**Comment by beto-rodriguez** [comment]
Sync the slider with the axis limits, that should work, similar to this sample: https://livecharts.dev/docs/Maui/2.0.0-beta.950/samples.axes.paging
